src/chapters/databases-part2/db-setup.rst

```diff
@@ -0,0 +1,101 @@
+Database Setup
+==============
+
+The Minesweeper game will use a combination of session cookies and database
+operations to store important information. Let's begin by getting the database
+file ready.
+
+Start With an Outline
+---------------------
+
+Before we write any code, we need to think about what data the game needs in
+order to work. In particular, we want to identify the information to save in
+the game's database. There are LOTS of possibilities! However, for our basic
+game, we want to start small. We can always refine the database structure
+later!
+
+Consider the data that needs to *persist*. For example, if a user leaves the
+game in the middle of play and comes back later, what needs to go into the
+database to keep a record of their progress?
+
+For starters, our database should save:
+
+#. Information about each mine hidden on the board. This includes a primary key
+   value as well as a location (row letter plus column number).
+#. Information about each individual cell on the board. This includes a primary
+   key, the cell's location, the number of mines in surrounding cells, whether
+   the user has clicked the cell, and whether the cell contains a mine.
+
+This outline suggests 2 tables for the database - one for a list of mines, and
+another for the cells on the game board.
+
+``db_setup.py``
+---------------
+
+Use the tools in Visual Studio code to create a new Python file at the same
+level as ``main.py``. Call the new file ``db_setup.py``. This will be a
+separate program, and its only job is to create the database file and add two
+tables.
+
+Follow along with the video to build this program.
+
+.. admonition:: Tip
+
+   The video player automatically sets the resolution. However, this choice
+   might not be good enough to clearly see all of the text.
+
+   If the code looks blurry on your screen, click the gear icon in the video
+   controls and increase the resolution to the maximum value.
+   
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube.com/embed/arIdYX7dBDU" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
+.. admonition:: Note
+
+   For our basic game, we might not use all of the columns in the tables yet.
+   However, they could be useful later, so there's no harm in adding them now.
+
+Video Summary
+-------------
+
```

Comment:
I started from scratch - just a note, typing in python db_setup.py throws an error for me, I forgot to try python3 db_setup.py, I just hit the run button.  I do think that at this point students should be able to actively debug but just wanted to make a note.

Reply:
Corrected in revised video.